{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2006-2400",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "vendor_name": "outgun",
            "product": {
              "product_data": [
                {
                  "product_name": "outgun",
                  "version": {
                    "version_data": [
                      {
                        "version_value": "1.0"
                      },
                      {
                        "version_value": "1.0.3"
                      },
                      {
                        "version_value": "1.0.3_bot_2"
                      }
                    ]
                  }
                }
              ]
            }
          }
        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "NVD-CWE-Other"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://aluigi.altervista.org/adv/outgunx-adv.txt"
        },
        {
          "url": "http://secunia.com/advisories/20098"
        },
        {
          "url": "http://securityreason.com/securityalert/898"
        },
        {
          "url": "http://www.osvdb.org/25488"
        },
        {
          "url": "http://www.securityfocus.com/archive/1/archive/1/433932/100/0/threaded"
        },
        {
          "url": "http://www.securityfocus.com/bid/17985"
        },
        {
          "url": "http://www.vupen.com/english/advisories/2006/1796"
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/26511"
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "The leetnet functions (leetnet/rudp.cpp) in Outgun 1.0.3 bot 2 and earlier allow remote attackers to cause a denial of service (game interruption) via large packets, which cause an exception to be thrown."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "operator": "OR",
        "cpe": [
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:outgun:outgun:1.0",
            "cpe23Uri": "cpe:2.3:a:outgun:outgun:1.0:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:outgun:outgun:1.0.3",
            "cpe23Uri": "cpe:2.3:a:outgun:outgun:1.0.3:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:outgun:outgun",
            "cpe23Uri": "cpe:2.3:a:outgun:outgun:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "1.0.3_bot_2"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "version": "2.0",
        "vectorString": "(AV:N/AC:L/Au:N/C:N/I:N/A:C)",
        "accessVector": "NETWORK",
        "accessComplexity": "LOW",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "COMPLETE",
        "baseScore": 7.8
      },
      "severity": "HIGH",
      "exploitabilityScore": 10.0,
      "impactScore": 6.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2006-05-16T01:02Z",
  "lastModifiedDate": "2017-07-20T01:31Z"
}